base/sparse/sparsematrix.jl

```diff
@@ -732,22 +732,22 @@ ctranspose{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}) = qftranspose(A, 1:A.n, Base.ConjFu
 ## fkeep! and children tril!, triu!, droptol!, dropzeros[!]
 
 """
-    fkeep!{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, f, other, trim::Bool = true)
+    fkeep!(A::AbstractSparseArray, f, other, trim::Bool = true)
 
 Keep elements of `A` for which test `f` returns `true`. `f`'s signature should be
 
-    f{Tv,Ti}(i::Ti, j::Ti, x::Tv, other::Any) -> Bool
+    f(i::Int, [j::Int,] x, other::Any) -> Bool
```

Comment:
No, I guess this could rigorously be written as `f{Tv<:T1, Ti<:Tv}(i::T1, j::T2, ...`, but people will probably understand without  inventing a new syntax.